lib/AST/Decl.cpp

```diff
@@ -4326,6 +4339,16 @@ bool ValueDecl::isAccessibleFrom(const DeclContext *useDC,
                      [&]() { return getFormalAccess(); });
 }
 
+ImportAccessLevel ValueDecl::getImportAccessFrom(const DeclContext *useDC) const {
+  ModuleDecl *Mod = getModuleContext();
+  if (useDC && useDC->getParentModule() != Mod) {
```

Comment:
Is it valid for `useDC` to be `nullptr` here? I'm wondering if it would make sense to assert instead.